File: CPHLogicServer/src/main/java/Book.java
public class Book {
    private String isbn;
    private String title;
    private String author;
    private String genre;
    private int totalCopies;
    private int availableCopies;

    public Book(String isbn, String title, String author, String genre, int totalCopies, int availableCopies) {
        this.isbn = isbn;
        this.title = title;
        this.author = author;
        this.genre = genre;
        this.totalCopies = totalCopies;
        this.availableCopies = availableCopies;
    }

    public String getIsbn() {
        return isbn;
    }
    public String getTitle() {
        return title;
    }
    public String getAuthor() {
        return author;
    }
    public String getGenre() {
        return genre;
    }
    public int getTotalCopies() {
        return totalCopies;
    }
    public int getAvailableCopies() {
        return availableCopies;
    }

    public void setAvailableCopies(int availableCopies) {
        this.availableCopies = availableCopies;
    }
}

File: PersistenceServer/Server/Entities/Book.cs
namespace Entities;

public class Book
{
    public int Id { get; set; }
    public required string ISBN { get; set; }
    public required string Author { get; set; }
    public required string Title { get; set; }
    public required string State { get; set; }
    public required List<Genre> Genre { get; set; }
    public required List<Loan> Loans { get; set; }
}
